internal/quickstart/show_sdk_instructions.go

```diff
@@ -156,7 +156,7 @@ func (m showSDKInstructionsModel) View() string {
 	}
 
 	if m.instructions == "" || m.environment == nil {
-		return m.spinner.View() + fmt.Sprintf(" Fetching %s SDK instructions...\n", m.displayName) + footerView(m.help.View(m.helpKeys), nil)
```

Comment:
Darn, I caught the one in the quickstart title but not this.

Reply:
Yeah, it's a bummer we don't have tests for the setup UI to catch stuff like this.